facade/servicetemplate.go

```diff
@@ -357,19 +335,19 @@ func (f *Facade) DeployService(ctx datastore.Context, poolID, parentID string, o
 		}
 	}
 
-	return f.deployService(ctx, tenantID, svc.ID, svc.DeploymentID, poolID, overwrite, svcDef)
+	var statusUpdater = func(status string) {}
```

Comment:
I believe we do call DeployService when we are creating services as a result of installing a zenpack.  MigrateService and UpdateService does not handle service creation.